Create Blog Post “how-to-redeem-a-coupon-code” (#1833)
Nexmo/deved-platform: content/blog/en/how-to-redeem-a-coupon-code.md

In this blog post, you will learn what a coupon code is, and how to redeem it to go forth and build amazing things. 

### What is a Coupon Code?

A coupon code is our way to encourage you to build amazing projects. Redeeming a coupon code will increase your account balance and enable you to use this balance for various projects. 

Typically our coupon codes are alphanumeric codes and may be given out as part of workshops, talks, or special events. 

These coupon codes will only be valid for a specific period of time, so be sure to redeem them quickly! 

### How Do I Redeem a Coupon Code?

In order to redeem your coupon code, you will need to create a [Vonage API account](https://dashboard.nexmo.com/sign-up). Once you have created your account and confirmed your email address, take a look at your [Dashboard](https://dashboard.nexmo.com/). 

![Screenshot of Vonage API Dashboard view.](/content/blog/how-to-redeem-a-coupon-code/dashboard.png "Screenshot of Vonage API Dashboard view.")

> On the left-hand side of your dashboard, you will see what your balance is. Typically, if you created a new account, you will see a balance of €2. \

Reviewer: This keeps showing up automatically in Netlify, I deleted it multiple times :(

> (**Note**: Your balance will be shown in Euro (€) regardless of your location). 

Reviewer: Both are correct :) I will just leave it as is :)

---
title: How to Redeem a Coupon Code
description: "In this blog post, you will learn what a coupon code is and how to

>   redeem it. "

Reviewer 2: ```suggestion
  redeem it."
```

Reviewer: Done

thumbnail: /content/blog/how-to-redeem-a-coupon-code/coupon-code.png
author: caroline-kerns
published: true
published_at: 2021-11-18T21:45:25.874Z
updated_at: 2021-11-18T21:45:25.907Z
category: tutorial
tags:

>   - coupons

Reviewer 2: We usually add 3 tags, do you think there are any more you can add?

Reviewer: I don't think so, I tried to find more yesterday and came up empty. The instructions said 1-3 tags and I've seen a few of our posts only have 1 tag as well, should be fine :)

comments: false
spotlight: false
redirect: ""
canonical: ""
outdated: false
replacement_url: ""
---

> Whether you attended one of our events, participated in a challenge, or spoke with one of our lovely [](https://developer.vonage.com/team)[DevRel Team Members](https://developer.vonage.com/team), you likely will have come across one of our coupon codes.

Reviewer 2: ```suggestion
Whether you attended one of our events, participated in a challenge, or spoke with one of our lovely [DevRel Team Members](https://developer.vonage.com/team), you likely will have come across one of our coupon codes.
```

Reviewer: Done

In this blog post, you will learn what a coupon code is, and how to redeem it to go forth and build amazing things. 

### What is a Coupon Code?

A coupon code is our way to encourage you to build amazing projects. Redeeming a coupon code will increase your account balance and enable you to use this balance for various projects. 

Typically our coupon codes are alphanumeric codes and may be given out as part of workshops, talks, or special events. 

These coupon codes will only be valid for a specific period of time, so be sure to redeem them quickly! 

> ### How do I redeem a coupon code? 

Reviewer 2: Please check our capitalisation for titles. https://capitalizemytitle.com/style/AP/

Reviewer: Done

In order to redeem your coupon code, you will need to create a [Vonage API account](https://dashboard.nexmo.com/sign-up). Once you have created your account and confirmed your email address, take a look at your [Dashboard](https://dashboard.nexmo.com/). 

![Screenshot of Vonage API Dashboard view.](/content/blog/how-to-redeem-a-coupon-code/dashboard.png "Screenshot of Vonage API Dashboard view.")

> On the left-hand side of your dashboard, you will see what your balance is. Typically, if you created a new account, your will see a balance of €2. \

Reviewer 2: ```suggestion
On the left-hand side of your dashboard, you will see what your balance is. Typically, if you created a new account, you will see a balance of €2.
```

Reviewer: Done

![Screenshot of extended profile dropdown menu with Coupons highlighted.](/content/blog/how-to-redeem-a-coupon-code/coupons.png "Screenshot of extended profile dropdown menu with Coupons highlighted.")

Next, click on your name on the left-hand side to expand the menu, then click on "Coupons". 

![Coupon redemption screenshot with an arrow pointing at coupon box.](/content/blog/how-to-redeem-a-coupon-code/addcode.png "Coupon redemption screenshot with an arrow pointing at coupon box.")

The coupon redemption will open. In this view, navigate to the text box, enter your coupon code, and click the "Submit" button.

![Coupon success screenshot with arrows pointing to success message and balance.](/content/blog/how-to-redeem-a-coupon-code/couponsucceeded.png "Coupon success screenshot with arrows pointing to success message and balance.")

If your coupon was successfully redeemed, you will see two things:

1. A small pop up with a green border will announce: "Coupon successfully applied".
2. Your balance will have increased by the coupon amount. 

*Congratulations, you have successfully redeemed your coupon code!*

> ### What if something went wrong? 

Reviewer 2: Please check our capitalisation for titles. https://capitalizemytitle.com/style/AP/
```suggestion
### What if Something Went Wrong? 
```

Reviewer: Done

![Coupon failed screenshot with arrow pointing at the error message.](/content/blog/how-to-redeem-a-coupon-code/couponfailedcouponnot-found.png "Coupon failed screenshot with arrow pointing at the error message.")

If your coupon redemption has failed, you will see a red border around the coupon code box with an error message stating why the coupon redemption has failed. In this case, double-check that you have entered the coupon code correctly and try again. 

If the redemption keeps failing, please message us on [Twitter](https://twitter.com/VonageDev) or send us a message in the [Community Slack Channel](https://developer.vonage.com/community/slack). Be sure to include the coupon code you are attempting to redeem and the error message you are seeing, so we can investigate (and resolve) the problem. 

### What's Next?

> Now it is up to you to build amazing projects! If you are brand-new to Vonage, take a look at our [tutorials](https://developer.vonage.com/tutorials). If you are looking for new ideas and inspirations, check out our [blogs](https://learn.vonage.com/). 

Reviewer 2: ```suggestion
Now it is up to you to build amazing projects! If you are brand-new to Vonage, take a look at our [tutorials](https://developer.vonage.com/tutorials). If you are looking for new ideas and inspirations, check out our [blog posts](https://learn.vonage.com/). 
```

Reviewer: Done

> We hope you feel inspired and we are looking forward to seeing what you build with your  Vonage balance. Tag us on [Twitter ](https://twitter.com/VonageDev)and share your projects in our [Community Slack Channel](https://developer.vonage.com/community/slack). If you love to create content around your projects, take a look at our [Developer Spotlight Program.](https://learn.vonage.com/spotlight/)

Reviewer 2: ```suggestion
We hope you feel inspired and we are looking forward to seeing what you build with your Vonage balance. Tag us on [Twitter ](https://twitter.com/VonageDev) and share your projects in our [Community Slack Channel](https://developer.vonage.com/community/slack). If you love to create content around your projects, take a look at our [Developer Spotlight Program.](https://learn.vonage.com/spotlight/)
```

Reviewer: Done

### Any Questions? 

> If you have any questions around coupons and redemptions, please don't hesitate to reach out to us via the [Community Slack Channel](https://developer.vonage.com/community/slack). 

Reviewer 2: Grammarly suggested changing around to about - up to you!
```suggestion
If you have any questions about coupons and redemptions, please don't hesitate to reach out to us via the [Community Slack Channel](https://developer.vonage.com/community/slack). 
```

Reviewer: Done
